ftfy/__init__.py

```diff
@@ -161,65 +186,73 @@ def fix_text(text,
     return ''.join(out)
 
 ftfy = fix_text
+fix_encoding = fixes.fix_encoding
```

Comment:
Sooo ... the `fix_encoding` function is being set globally in this file, but is not actually used in this file (I guess it's here so people can import it from the top level?).  Its deprecated name `fix_text_encoding` is imported directly, but is not actually used anywhere (it was previously used in the even-more-deprecated `fix_bad_encoding`); elsewhere in this file, `fix_text_encoding` is referenced as `fixes.fix_text_encoding`.  Additionally, three of the functions defined in this file take an *argument* called `fix_encoding` (not to be confused with this global variable of the same name).

This must be at least a little bit simplifiable.